nautobot/extras/templates/extras/job_approval_request.html

```diff
@@ -122,7 +122,7 @@ <h1>{{ object.name }}</h1>
                             >
                                 <i class="mdi mdi-close"></i> Deny
                             </button>
-                            <a href="{% url 'extras:scheduledjob_approval_queue_list' %}" class="btn btn-default">
+                            <a href="extras/jobs/approval-queue" class="btn btn-default">
```

Comment:
This change is a bad code smell. We should still be able to use `{% url ... %}` here.